dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/p2p/categories/general/P2pGeneral.kt

```diff
@@ -1,25 +1,25 @@
 package org.dataland.datalandbackend.model.p2p.categories.general
 
 import com.fasterxml.jackson.annotation.JsonProperty
-import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pClimateTargets
-import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pEmissionsPlanning
+import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGeneralClimateTargets
+import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGeneralEmissionsPlanning
 import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGeneralGeneral
-import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGovernance
-import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pInvestmentPlanning
+import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGeneralGovernance
+import org.dataland.datalandbackend.model.p2p.categories.general.subcategories.P2pGeneralInvestmentPlanning
 
 /**
  * --- API model ---
- * Fields of the P2P questionnaire regarding general information applicable to companies of all sectors.
- */
+ * Fields of the category "General" of the p2p framework.
+*/
 data class P2pGeneral(
     @field:JsonProperty(required = true)
     val general: P2pGeneralGeneral,
 
-    val governance: P2pGovernance? = null,
+    val governance: P2pGeneralGovernance? = null,
```

Comment:
This change is to be conform with naming conventions in other frameworks I assume?

Reply:
Would be my understanding at least.